src/tests/writeDebugSvg.ts

```diff
@@ -0,0 +1,25 @@
+import { writeFileSync, mkdirSync } from 'node:fs';
+
+import { SVGPathData } from '../index.js';
+
+let i = 0;
+
+export function writeDebugToSvg(input: SVGPathData, source: string) {
```

Comment:
i should not have to commited this, unless its ok, its useful little script that generates svg files from tests for final visual inspection